bindings/go/rsa/signing/handler/internal/dn/dn.go

```diff
@@ -0,0 +1,117 @@
+// Package dn provides utilities for parsing and matching X.509 Distinguished Names (DNs).
```

Comment:
Do... we... need all this? https://pkg.go.dev/crypto/x509 surely has most of this covered?